File: java/Dpr.java
/*SAya Amelia Zalfa Juianti dengan NIM 2203999 mengerjakan LP1 untuk keberkahannya maka saya
tisak melakukan kecurangan sesuai dengan apa yang telah dispesifikasikan. Aamiin*/
public class Dpr{
	//private attributes
	private String id;
	private String nama;
	private String bidang;
	private String partai;

	public Dpr(){
		this.id = "";
		this.nama = "";
		this.bidang = "";
		this.partai = "";
	}

	public Dpr(String id, String nama, String bidang, String partai){
		this.id = id;
		this.nama = nama;
		this.bidang = bidang;
		this.partai = partai;
	}

	/*Getter and Setter */

	//Get id

	public String getId(){
		return this.id;
	}

	//set id

	public void setId(String id){
		this.id = id;
	}

	//Get nama
	public String getNama(){
		return this.nama;
	}

	//set nama
	public void setNama(String nama){
		this.nama = nama;
	}

	//get bidang
	public String getBidang(){
		return this.bidang;
	}

	//set bidang
	public void setBidang(String bidang){
		this.bidang = bidang;
	}

	//get partai
	public String getPartai(){
		return this.partai;
	}

	//set partai
	public void setPartai(String partai){
		this.partai = partai;
	}


}

File: python/Dpr.py
#SAya Amelia Zalfa Juianti dengan NIM 2203999 mengerjakan LP1 untuk keberkahannya maka saya
#tisak melakukan kecurangan sesuai dengan apa yang telah dispesifikasikan. Aamiin
#class declaration
class Dpr:
    __id = -1
    __nama = ""
	__bidang = ""
    __partai = ""

	def __init__(self, id = 0, nama = "", bidang = "", partai = ""):
		self.__id = id
		self.__nama = nama
		self.__bidang = bidang
		self.__partai = partai

	#Getter and Setter

		#get ID
		def get_id(self):
			return self.__id
		#set id
		def set_id(self, id):
			self.__id = id

		#get name
			def get_nama(self):
		#set name
			def set_nama(self, nama):
				self.__nama = nama
		
		#get bidang
			def get_bidang(self):
		#set bidang
			def set_bidang(self, bidang):
				self.__bidang = bidang
		
		#get partai
			def get_partai(self):
		#set partai
			def set_partai(self, partai):
				self.__partai = partai
			
				
				    
